Altis_Life.Altis/core/init.sqf

```diff
@@ -179,6 +179,16 @@ life_hideoutBuildings = [];
     false
 } count ["gang_area_1","gang_area_2","gang_area_3"];
 
+// -- Disable commanding view
+if (LIFE_SETTINGS(getNumber,"commandview") isEqualTo 1) then {
+    addMissionEventHandler ["EachFrame",{
+        if(cameraView isEqualTo "GROUP") then {
+            player switchCamera "EXTERNAL";
+            hint [localize "STR_NOTF_COMMANDVIEW"];
```

Comment:
`STR_NOTF_COMMANDVIEW` will need to be added to the stringtable. Could you make it follow the same case as the others?